packages/@webex/plugin-cc/src/cc.ts

```diff
@@ -136,4 +136,26 @@ export default class ContactCenter extends WebexPlugin implements IContactCenter
       return Promise.reject(error);
     }
   }
+
+  /**
+   * This is used for agent logout.
+   * @param options
+   * @returns Promise<LogoutSuccess>
+   * @throws Error
+   */
+  public async stationLogout(options: {logoutReason: string}): Promise<LogoutSuccess> {
+    try {
+      const response = await this.agent.stationLogout(options);
+
+      if (this.webRTCCalling) {
+        this.webRTCCalling.deregisterWebCallingLine();
+      }
+
+      return response;
```

Comment:
Add a log for the success case

Reply:
`this.agent.stationLogout` is already adding the success log. Adding here will add it twice.

Reply:
@rarajes2 , In that case, let's remove the logs from catch

Reply:
Done